packages/host/ember-cli-build.js

```diff
@@ -74,6 +74,9 @@ module.exports = function (defaults) {
           plugins: [
             new GlimmerScopedCSSWebpackPlugin(),
             new MonacoWebpackPlugin(),
+            new webpack.DefinePlugin({
+              'process.env.NODE_DEBUG': JSON.stringify(process.env.NODE_DEBUG),
+            }),
```

Comment:
In my attempts with this I also tried a workaround for `process.env.NODE_DEBUG` but the underlying problem was that `process.env` just didn’t exist, so the next failure was another attempt to use it. It seems like something is weird with the build environment that it wouldn’t exist anymore.

I was able to see this failure locally, maybe if you check out a fresh repository you can too so you don’t have to debug via CI?